engine/table/src/test/java/io/deephaven/engine/table/impl/TestConcurrentInstantiation.java

```diff
@@ -405,66 +405,57 @@ public void testWhereDynamicIndexedBoth() throws ExecutionException, Interrupted
 
     private void testWhereDynamicInternal(final boolean sourceIndexed, final boolean setIndexed)
             throws ExecutionException, InterruptedException, TimeoutException {
-        final QueryTable table = TstUtils.testRefreshingTable(i(2, 4, 6).toTracking(),
-                col("x", 1, 2, 3), col("y", "a", "b", "c"), col("z", true, false, true));
+        final QueryTable table = TstUtils.testRefreshingTable(i(2, 4, 6, 8, 10).toTracking(),
```

Comment:
I increased the size of the source table to get above the threshold for data index usage.